src/index.html

```diff
@@ -17,6 +25,87 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="/"
+        class="logo-link"
+      >
+        <img
+          src="images/logo.png"
+          alt="MOYO Electronics Brand Logo'"
```

Comment:
The `alt` attribute should describe the image content, but avoid using special characters like single quotes at the end unless they are part of the actual description. It should be `alt="MOYO Electronics Brand Logo"`.